examples/cloud-operations/network-dashboard/cloud-function/requirements.txt

```diff
@@ -0,0 +1,8 @@
+regex
```

Comment:
I'd pin the all the packages listed here to specific versions

Reply:
Done in my 2nd commit, thanks.